holmes/plugins/toolsets/datadog/toolset_datadog_metrics.py

```diff
@@ -404,6 +466,92 @@ def get_parameterized_one_liner(self, params) -> str:
         return "Get Datadog metric metadata"
 
 
+class ListMetricTags(BaseDatadogMetricsTool):
+    def __init__(self, toolset: "DatadogMetricsToolset"):
+        super().__init__(
+            name="list_datadog_metric_tags",
+            description="List all available tags and aggregations for a specific metric. This helps in building queries by showing what dimensions are available for filtering.",
+            parameters={
+                "metric_name": ToolParameter(
+                    description="The name of the metric to get tags for (e.g., 'system.cpu.user', 'container.memory.usage')",
+                    type="string",
+                    required=True,
+                ),
+            },
+            toolset=toolset,
+        )
+
+    def _invoke(self, params: Any) -> StructuredToolResult:
+        if not self.toolset.dd_config:
+            return StructuredToolResult(
+                status=ToolResultStatus.ERROR,
+                error=TOOLSET_CONFIG_MISSING_ERROR,
+                params=params,
+            )
+
+        url = None
+        query_params = None
+
+        try:
+            metric_name = get_param_or_raise(params, "metric_name")
+
+            url = f"{self.toolset.dd_config.site_api_url}/api/v2/metrics/{metric_name}/active-configurations"
+            headers = get_headers(self.toolset.dd_config)
+
+            data = execute_datadog_http_request(
+                url=url,
+                headers=headers,
+                timeout=self.toolset.dd_config.request_timeout,
+                method="GET",
+                payload_or_params={},
+            )
+
+            return StructuredToolResult(
+                status=ToolResultStatus.SUCCESS,
+                data=data,
```

Comment:
1. Do we really want to return the raw response and not only "active_tags" from it?
2. No reason to check for no_data case?